controllers/upgrade_handlers.go

```diff
@@ -148,6 +150,11 @@ func (u *UpgHandler) PrePivot(ctx context.Context, ibu *lcav1alpha1.ImageBasedUp
 	}
 
 	u.Log.Info("Writing extra-manifests into new stateroot")
```

Comment:
Can we add some comments here on  this mechanism.

Reply:
Updated